ProcessLib/Utils/ProcessUtils.cpp

```diff
@@ -105,4 +105,24 @@ std::vector<std::reference_wrapper<ProcessVariable>> findProcessVariables(
 
     return vars;
 }
+
+ParameterBase* findParameterByName(
+    std::string const& parameter_name,
+    std::vector<std::unique_ptr<ParameterBase>> const& parameters)
+{
+    // Find corresponding parameter by name.
+    auto const it = std::find_if(
+        parameters.cbegin(), parameters.cend(),
+        [&parameter_name](std::unique_ptr<ParameterBase> const& p) {
+            return p->name == parameter_name;
+        });
+
+    if (it == parameters.end())
+    {
+        return nullptr;
```

Comment:
Is this a usual case? Maybe add also some debug output to detect if there is an error?

Reply:
The nullptr is checked later by the caller, which issues a warning or whatever.